packages/core/tests/api/NotionUser.test.ts

```diff
@@ -287,15 +287,15 @@ describe('NotionUser', () => {
         token: 'token',
         user_id: 'user_root_1'
       });
-  
+
       const initializeCacheForSpecificDataMock = jest
         .spyOn(NotionData.prototype, 'initializeCacheForSpecificData')
         .mockImplementationOnce(() => {
           return {} as any;
         });
-  
+
       const page_map = await notion_user.getPagesById([block_1_id, block_2_id]);
-      expect(initializeCacheForSpecificDataMock).toHaveBeenCalledTimes(2);
+      expect(initializeCacheForSpecificDataMock).toHaveBeenCalledTimes(3);
```

Comment:
i figured it would be called once more since i added a call to this.initializeCacheForSpecificData() on line 234? I'm not sure why it was 2 to begin with and not just 1.

Reply:
It was 2 due to line 297 `const page_map = await notion_user.getPagesById([block_1_id, block_2_id]);`. Here two block ids are given, one of them is `page` the other being a cvp, that's why it should be called twice to initialize cache for those two only.

Reply:
This should be changed back to 2 as before.